core/domain/obj_services.py

```diff
@@ -71,3 +74,10 @@ def get_all_object_editor_js_templates():
             object_editors_js += obj_cls.get_editor_js_template()
 
     return object_editors_js
+
+
+def get_default_object_values():
+    """Returns a dictionary containing the default object values."""
+
+    return json.loads(
```

Comment:
Should this be cached? How often is this going to be hit? It seems like a lot for loading a file everytime.